src/page-components/PageFooter/useLocales/qya-AA.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Social": "crwdns9523:0crwdne9523:0",
```

Comment:
What does `crwdns9523:0crwdne9523:0` mean? These result might not suit for the `i18n` algorithm